scripts/tests.sh

```diff
@@ -28,8 +28,21 @@
 
 set -e
 
-# There is an implicit assumption here that we HAVE to run from root directory.
-REPO_ROOT=$(pwd)
+REPO_ROOT="$(dirname "$0")"/..
+
+ # Compile all files in std and examples.
+
+for f in "$REPO_ROOT"/std/*.sol
+do
+    echo "Compiling $f..."
+    set +e
+    output=$("$REPO_ROOT"/build/solc/solc "$f" 2>&1)
+    failed=$?
+    output=$(echo "$output" | grep -v 'pre-release')
```

Comment:
Probably add a comment here that we must remove the pre-release version warning from the output.